flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -1005,58 +1001,6 @@ private void submitNoOpInvokableTask(
 		tmGateway.submitTask(tdd, jobMasterGateway.getFencingToken(), timeout).join();
 	}
 
-	/**
-	 * This tests makes sure that duplicate JobMaster gained leadership messages are filtered out
-	 * by the TaskExecutor. See FLINK-7526.
-	 */
-	@Test
-	public void testFilterOutDuplicateJobMasterRegistrations() throws Exception {
```

Comment:
where is this now tested?

Reply:
The reason I removed this test case is that the API of the `JobTable` no longer allows to create two `JobManagerConnections`. Hence the previous behaviour is now tested via `DefaultJobTableTest#getOrCreateJob_RegisteredJob_WillReturnRegisteredJob` and `#connectJob_Connected_Fails`.